src/unix/solarish/illumos.rs

```diff
@@ -58,12 +184,68 @@ pub const SOL_FILTER: ::c_int = 0xfffc;
 
 pub const MADV_PURGE: ::c_int = 9;
 
-pub const POSIX_FADV_NORMAL: ::c_int = 0;
-pub const POSIX_FADV_RANDOM: ::c_int = 1;
-pub const POSIX_FADV_SEQUENTIAL: ::c_int = 2;
-pub const POSIX_FADV_WILLNEED: ::c_int = 3;
-pub const POSIX_FADV_DONTNEED: ::c_int = 4;
-pub const POSIX_FADV_NOREUSE: ::c_int = 5;
```

Comment:
It looks like these, plus POLLRDHUP above, were accidentally deleted maybe in a bad merge. I'll roll a fix in for these in #3999.

Reply:
Sorry about that.